packages/dso-toolkit/src/components/button/button.mixins.scss

```diff
@@ -20,19 +20,21 @@
 }
 
 @mixin _shared($base-styling, $modifiers, $defaults-map, $hover-map, $active-map, $disabled-map, $icon-only) {
+  @include _shared-variant($defaults-map);
+
   @if ($base-styling) {
     @include base($modifiers);
   }
 
-  @include _shared-variant($defaults-map);
-
-  border-width: 1px;
-  border-style: solid;
-  border-radius: button-variables.$border-radius;
-  line-height: typography.$line-height-base;
-  min-inline-size: button-variables.$min-inline-size;
-  padding-block: button-variables.$block-padding;
-  padding-inline: button-variables.$inline-padding;
+  & {
```

Comment:
Ik zal er morgen ook even naar kijken, maar ik begrijp nog niet zo goed waarom de `& {}` truuk IN een mixin nodig is. Waarom plaats je deze niet op de plek waar de mixin wordt gebruikt?

Ik denkt dat de uitgangspositie moet zijn: `& {}` zo hoog mogelijk plaatsen. Waar "hoog" het begin van de stylesheet nesting is, en "diep" het einde van de recursieve mixin calls.

Ik meen mij te herinneren dat ik deze stelling al eens had gesteld, hebben we hier over gesproken?

Reply:
Daar ben ik nog mee bezig; donderdag paar dingen geprobeerd, maar die leveren toch weer deprecation meldingen op. Morgen verder.

Reply:
Ik heb de indruk dat ik qua oplossingen in herhaling begin te vallen, en daarbij een andere uitkomst verwacht (het begin van krankzinnigheid).. Ik heb er een hekel aan een beroep op je te doen tijdens je vakantie, maar hoe zie jij dit voor je?

dit werkt i.i.g. niet bij mij: 
`& {}` weg uit mixin + mixin aanroepen met `& { @include naam() }`. Als dat is wat je bedoelt, dan werkt dat niet, tenzij er nog wat ontbreekt.

Reply:
Ok, dat is precies wat ik bedoelde! Dus als dat niet werkte, dan is dat ook meteen het antwoord 🎉